rtnl/neigh.go

```diff
@@ -0,0 +1,33 @@
+// github.com/jsimonetti/rtnetlink/rtnl
+package rtnl
+
+import (
+	"github.com/jsimonetti/rtnetlink"
+	"net"
+)
+
+func Neighbours(conn *rtnetlink.Conn, ifc *net.Interface, family int) ([]rtnetlink.NeighMessage, error) {
```

Comment:
nit: `NeighbourList` ? (I'm also confused by the fact that `ip` uses both American and British spelling in the man page)

Reply:
Also same thing as above for the return type